examples/advanced/experiment-tracking/tensorboard/README.md

```diff
@@ -1,14 +1,43 @@
-# Hello PyTorch with Tensorboard Streaming
+# Hello PyTorch with TensorBoard Streaming
 
 Example of using [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) to train an image classifier
 using federated averaging ([FedAvg](https://arxiv.org/abs/1602.05629)) and [PyTorch](https://pytorch.org/)
-as the deep learning training framework.
+as the deep learning training framework with **TensorBoard experiment tracking**.
 
-This example also highlights the TensorBoard streaming capability from the clients to the server.
+This example demonstrates the **Recipe API** for easily adding TensorBoard streaming to FL training jobs.
 
 > **_NOTE:_** This example uses the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset and will load its data within the trainer code.
 
-### 1. Install requirements 
+## What's New: Recipe API + Experiment Tracking
```

Comment:
we should just explain this example in README, maybe we do not need to compare with old versions and say what's new?